Feature/WebEngine support (#187)
smartdevicelink/sdl_hmi_integration_guidelines: docs/BasicCommunication/OnAppPropertiesChange/index.md

## OnAppPropertiesChange

Type
: Notification

Sender
: SDL

Purpose
: Inform HMI about app properties change

!!! must

> SDL must notify HMI about webEngine app properties change (e.g. after PTU with the appropriate parameters for the web app).

Reviewer: ```suggestion
SDL must notify the HMI when a web engine app's properties change, such as after a PTU with the appropriate parameters for the web app.
```

!!!

#### Parameters
|Name|Type|Mandatory|Additional|
|:---|:---|:--------|:---------|
|properties|[Common.AppProperties](../../common/structs/#appproperties)|true||

### Example Notification
```json
{
  "id" : 47,
  "jsonrpc" : "2.0",
  "method" : "BasicCommunication.OnAppPropertiesChange",
  "params" :
  {
    "properties" : {

>       "nicknames": {"Hello Sdl1"}, 

Reviewer: ```suggestion
      "nicknames": ["Hello Sdl1"],
```
